jobs/upload-brew.Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+import org.yaml.snakeyaml.Yaml;
+
+def pipeutils, pipecfg, libcloud, uploading, jenkins_agent_image_tag
```

Comment:
Let's do the same thing as https://github.com/coreos/fedora-coreos-pipeline/commit/e1fdff84e42cd2b6c91bac176b180ebef9a571a3 here for consistency.